cypress/e2e/models/migration/applicationinventory/analysis.ts

```diff
@@ -112,6 +113,7 @@ export class Analysis extends Application {
         information?: number;
         total?: number;
     };
+    numberOfRules?: { [id: string]: Number };
```

Comment:
```suggestion
    numberOfRules?: { [id: string]: number };
```

Reply:
This is more of a personal suggesstion but I'd like to know wdyt about it:

The name `numberOfrules` sounds like it contains a `number` type, when creating a map conventios suggest to use more descriptive names by using the words `By` or `To`  `numberByRule`, `ruleToQuantity`.

Here are some examples: [1](https://medium.com/social-tables-tech/naming-maps-in-es6-b0989b390ddf), [2](https://stackoverflow.com/questions/217614/whats-a-good-naming-convention-for-a-lookup-map-hash), [3](https://softwareas.com/naming-conventions-for-key-value-maps/)

But, as I mentioned, since we don't have a style guide defined, this is personal taste and can be left as it is